api/test/trace/defaultspan_test.cc

```diff
@@ -0,0 +1,20 @@
+#include "opentelemetry/trace/default_span.h"
```

Comment:
Can this be named `default_span_test` instead of `defaultspan_test`?

Reply:
Sure